selfdrive/car/mazda/values.py

```diff
@@ -0,0 +1,69 @@
+from selfdrive.car import dbc_dict
+from cereal import car
+Ecu = car.CarParams.Ecu
+
+
+# Steer torque limits
+
+class SteerLimitParams:
+  STEER_MAX = 600                 # max_steer 2048
+  STEER_STEP = 1                  # how often we update the steer cmd
+  STEER_DELTA_UP = 10             # torque increase per refresh
+  STEER_DELTA_DOWN = 20           # torque decrease per refresh
+  STEER_DRIVER_ALLOWANCE = 15     # allowed driver torque before start limiting
+  STEER_DRIVER_MULTIPLIER = 1     # weight driver torque heavily
+  STEER_DRIVER_FACTOR = 1         # from dbc
+
+class CAR:
+#  CX3 = "Mazda CX-3"
```

Comment:
Seems like these are supported, but just treated as a CX5. Either break them out into separate cars, or delete these commented lines.

Reply:
They all have identical CAN buses and control, and even overlapping fingerprints. Many of these were added recently as more Mazda cars came online. I was planning on adding blocks for each car but hasn't been a priority so far